Makefile

```diff
@@ -55,6 +55,34 @@ test-ci:
 	@go test $(EXCLUDED_TEST_PACKAGES) -parallel $(PARALLEL_TESTS) -timeout $(TEST_TIMEOUT)
 	@echo "Tests complete!"
 
+run-localnet-linux/arm64:
+	@echo "Running localnet..."
+	@docker run -p 6006:6006 --rm -it -d --platform linux/arm64 --name rippled_standalone --volume $(PWD)/.ci-config:/etc/opt/ripple/ --entrypoint bash rippleci/rippled:2.3.0 -c 'rippled -a'
```

Comment:
Let's use the 2.3.1 image which has some bug fixes.
Is there a way to make the image version dynamic in our tests? For example if we want to test DeepFreeze, it will only be available in the `develop` branch or later in the 2.4.x.

Reply:
Just used the same as in xrpl.js